openapi/sms/v1/instructions.md

```diff
@@ -0,0 +1,244 @@
+
+```
+
+A partner service requires the following endpoints to support client onboarding:
+
+---
+
+- #### [Create/Update Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/get)
+
+##### Request Schema:
+
+```yaml json_schema
+$ref: '../partner-service.yaml#/components/schemas/CreateOrUpdateClientConfigurationRequest'
+```
+
+##### Response Schema:
+
+```yaml json_schema
+$ref: '../partner-service.yaml#/components/schemas/CreateOrUpdateClientConfigurationResponse'
+```
+
+---
+
+- [Get Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/get)
+
+##### Response Schema:
+
+```yaml json_schema
+$ref: '../partner-service.yaml#/components/schemas/GetClientConfigurationResponse'
+```
+
+---
+
+- [Delete Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/delete)
+
+---
+
+### Outbound Messaging and Reporting
+
+#### Overview
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each SMS campaign trigger
+    Client-->Emarsys: Triggers Campaign
+    Emarsys->>Partner: Outbound Messages
+    Partner->>Vendor: Outbound SMS Messages
+    loop Each Delivery Report
+      Vendor->>Partner: SMS Delivery Receipt
+      Partner->>Emarsys: Message Delivery Report
+      Emarsys->>Client: Message Reporting
+      Note left of Client: User views reports via UI
+    end
+  end
+```
+
+#### Outbound Messages
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each SMS campaign trigger
+    Client-->Emarsys: Triggers Campaign
+    Emarsys->>Partner: Outbound Messages
+    Partner->>Vendor: Outbound SMS Messages
+  end
+```
+
+- [Create Outbound Message Batch](../partner-service.yaml/paths/~1clients~1{clientId}~1messages/post)
+  - Request Payload
+    ```yaml json_schema
+    $ref: '../partner-service.yaml#/components/schemas/OutboundMessageBatch'
+    ```
+
+#### Delivery Reports
+
+Delivery reports are communicated via a callback to Emarsys SMS:
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each Delivery Report
+      Vendor->>Partner: SMS Delivery Receipt
+      Partner->>Emarsys: Message Delivery Report
+      Emarsys->>Client: Message Reporting
+      Note left of Client: User views reports via UI
+  end
+```
+
+---
+
+- [Create Delivery Report Batch](../partner-callbacks.yaml/paths/~1clients~1{clientId}~1deliveryReports)
+
+  ***
+
+  - ##### Request
+    ***
+    - Delivery Report:
+      ```yaml json_schema
+      $ref: '../partner-callbacks.yaml#/components/schemas/DeliveryReport'
```

Comment:
```suggestion
      $ref: './partner-callbacks.yaml#/components/schemas/DeliveryReport'
```

Reply:
we navigate to the parent directory here because in stoplight markdown files are places in a `docs` directory (within the root) whereas the OpenAPI yaml files are directly in the root